src/DynamoCoreWpf/Views/Search/LibraryView.xaml.cs

```diff
@@ -97,8 +97,12 @@ private void OnMemberMouseEnter(object sender, MouseEventArgs e)
             }
 
             FrameworkElement fromSender = sender as FrameworkElement;
-            libraryToolTipPopup.PlacementTarget = fromSender;
-            libraryToolTipPopup.SetDataContext(fromSender.DataContext);
+            var memberVM = fromSender.DataContext as NodeSearchElementViewModel;
```

Comment:
I think we don't need it, because it's written in XAML.
I.e.

```xaml
<uicontrols:LibraryToolTipPopup x:Name="libraryToolTipPopup"
                                        StaysOpen="True"
                                        AllowsTransparency="True"
                                        MouseLeave="OnPopupMouseLeave" />
```